moxin-frontend/src/chat/chat_history.rs

```diff
@@ -0,0 +1,228 @@
+use crate::data::{chat::ChatID, store::Store};
+
+use chrono::{DateTime, Local, NaiveDateTime, TimeZone, Utc};
+
+use makepad_widgets::*;
+
+live_design! {
+    import makepad_widgets::base::*;
+    import makepad_widgets::theme_desktop_dark::*;
+
+    import crate::shared::styles::*;
+    import makepad_draw::shader::std::*;
+
+    ChatCard = {{ChatCard}} {
+        content = <RoundedView> {
```

Comment:
Just a tip for the future, if the reason you're creating a `content = <RoundedView>` is that you want `ChatCard` to have the look/behaviour of `RoundedView`, you can do so directly by 'inhering':
`ChatCard = {{ChatCard}}<RoundedVIew> { ... }` in that way you don't have to deal with accessing `self.view(id!(content));`.
I wouldn't bother with changing this now though.

Reply:
thats exactly it yeah, thanks.